docker/run_celery.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+while ! nc -z rabbit 5672; do echo "Waiting for RabbitMQ... \n"; sleep 3; done
```

Comment:
If you add an `DEPENDENT_SERVICES: "RABBIT"` environment variable then I think it should